src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
What if we have another exception with some other service involved? We'll have a misguiding log that `AuthernticationServiceimpl` threw some exception when in fact it has nothing to do with it. We pass a class where this logger is located to the parameter, not what the exception is, or about what is this log
```suggestion
    private static final Logger logger = LogManager.getLogger(Main.class);
```